pkg/operator/ceph/csi/template/rbd/csi-rbdplugin-provisioner-dep.yaml

```diff
@@ -21,6 +21,14 @@ spec:
       {{ if .ProvisionerPriorityClassName }}
       priorityClassName: {{ .ProvisionerPriorityClassName }}
       {{ end }}
+      initContainers:
+        - name: create-logrotate.d-config
+          image: {{ .CSIPluginImage }}
+          command:
+          - touch /etc/logrotate.d/csi
```

Comment:
need to add this content to file
```
/dev/*.log {
  rotate 3
  daily
  size 1M
  copytruncate
  start 1
}
```

any best ways?

Reply:
As an example, look at how the config is modified for the ceph containers [here](https://github.com/rook/rook/blob/63bbbe4de8455cea73be836d0c8c442092265d9b/pkg/operator/ceph/controller/spec.go#L103-L130).

Reply:
Figured that out with the container itself

Reply:
You've already suggested putting it in a configmap and mounting it into the pod. That seems reasonable to me, though I do worry about what happens if users modify config values after deployment. IIRC, configmap changes don't appear in pods until the pod is restarted. 

In order to ensure the configs are updated when users change values, setting up the config into an emptyDir in an init container might be the most straightforward, even if it's clunky.